upload_metadata.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3
+
+import argparse
+import configparser
+# import re
+import pandas as pd
+
+from iridauploader import api, model
+
+def init_parser():
+    '''
+    Specify command line arguments for automatic upload to irida
+    Returns command line parser with inputs
+    '''
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '-c',
+        '--config',
+        required=True,
+        help='IRIDA Uploader config file to parse'
+    )
+    parser.add_argument(
+        '-m',
+        '--metadata_csv',
+        required=True,
+        help='Metadata csv for upload containing matching ids to SampleList.csv in the FIRST column and a project id in the SECOND'
```

Comment:
Same thing as in the readme, the two columns just need to exist in the input file

Reply:
Fixed in https://github.com/phac-nml/irida-metadata-uploader/pull/1/commits/c3e2baf188e14445c9a755235a1273664e7a9081